### PR TITLE
DEVEX-2067: Add warning about rerunning download command for missing/modified files

### PR DESCRIPTION
**Pull request by wormsik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,12 +72,13 @@ and you will get a brief summary of the status the downloads:
 21.6 MB/sec	1184/27078 MB	18/327 Parts Downloaded and written to disk
 ```
 
-
+Once the download is finished, it is advised to perform an inspection of the downloaded files to ensure their integrity:
 ```
 dx-download-agent inspect exome_bams_manifest.json.bz2
 ```
+This command will perfrom an inspection of the parts of files currently downloaded to disk and ensure that their MD5sums match the manifest. If a file is missing or an MD5sum does not match, the download agent will report the affected files. After the inspection is complete, you can issue a `download` command again to resolve the issues.
```
Comment (emiloslavsky) [inline README.md:79]:
```suggestion
This command will perform an inspection of the parts of files currently downloaded to disk and ensure that their MD5sums match the manifest. If a file is missing or an MD5sum does not match, the download agent will report the affected files. After the inspection is complete, you can issue a `download` command again to resolve the issues.
```

File: README.md [hunk 2]
```diff
@@ -72,12 +72,13 @@ and you will get a brief summary of the status the downloads:
 21.6 MB/sec	1184/27078 MB	18/327 Parts Downloaded and written to disk
 ```
 
-
+Once the download is finished, it is advised to perform an inspection of the downloaded files to ensure their integrity:
```
[the remark this answers is not in the document]
Comment (wormsik) [reply README.md]: yes, you are right...the checksums are already checked during download...I re-phrased it and moved the warning right below download command description as both `progress` and `inspect` are generally much less relevant and can be skipped

File: README.md [hunk 3]
```diff
@@ -60,6 +60,8 @@ reflects not only network bandwidth, but also the IO capability of your machine.
 
 A download log contains more detailed information about the download should an error occur.  If an error does occur and you do not understand how to deal with it, please contact `support@dnanexus.com` with the log file attached and we will assist you.
 
+**WARNING:** Please note, that simple rerunning `dx-download-agent download` command will NOT redownload any manually moved, deleted or modified downloaded files! It is necessary to run `dx-download-agent inspect` command (describe below) first to detect the changes and mark them for redownload. More details can be found in the section [Moving downloaded files](#moving-downloaded-files).
```
Comment (emiloslavsky) [inline README.md:63]:
```suggestion
Please note that rerunning `dx-download-agent download` command will NOT re-download any previously downloaded files that were subsequently moved, deleted or modified.  Please run `dx-download-agent inspect` (described below) to detect any changes to previously downloaded files and mark them for re-download.  See [Moving downloaded files](#moving-downloaded-files) for more details.
```

File: README.md [hunk 4]
```diff
@@ -72,12 +74,11 @@ and you will get a brief summary of the status the downloads:
...
 ```
 
-
+To check the integrity of the downloaded files, you can run
 ```
 dx-download-agent inspect exome_bams_manifest.json.bz2
 ```
-
-This command will perfrom an inspection of the parts of files currently downloaded to disk and ensure that their MD5sums match the manifest.  If a file is missing or an MD5sum does not match, the download agent will report the affected files.  After the inspection is complete, you can issue a `download` command again to resolve the issues.
+This command will perform an inspection of the files and ensure that their MD5sums match the manifest. If a file is missing or an MD5sum does not match, the download agent will report the affected files and you can run a `download` command again to resolve the issues.
```
Comment (emiloslavsky) [inline README.md:81]:
```suggestion
This command will perform an inspection of the files and ensure that their MD5sums match the manifest. If a file is missing or an MD5sum does not match, the download agent will report the affected files and you can then run `dx-download-agent download` again to re-download the affected files.
```

**Review by emiloslavsky — APPROVED** [review]
LGTM with minor verbiage fixes

**Pull request merged**